dev/devicelab/manifest.yaml

```diff
@@ -115,6 +115,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["has-android-device"]
 
+  import_test:
```

Comment:
This will create a new column on the build table and we're running out of horizontal screen space. Let's instead roll this test into the `e2e` test.